base_layer/core/src/transactions/transaction/output_features.rs

```diff
@@ -43,9 +44,26 @@ use crate::{
     },
 };
 
+#[derive(Debug, Clone, Hash, PartialEq, Deserialize, Serialize, Eq)]
+#[repr(u8)]
+pub enum OutputFeaturesVersion {
+    DIBBLER = 0,
+}
+
```

Comment:
I think we need an `OutputFeaturesVersion::GetCurrentVersion()` implementation here.